packages/react/src/components/Grid/FlexGrid.stories.js

```diff
@@ -1,6 +1,6 @@
 import './FlexGrid.stories.scss';
 import React from 'react';
-import { FlexGrid, Row, Column } from '../';
+import { FlexGrid, Row, Column } from './index';
```

Comment:
index is automatically inferred so this can be updated to:
```suggestion
import { FlexGrid, Row, Column } from './';
```